.changeset/nine-pigs-whisper.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
```

Comment:
Every time when this package is update also `preview-middleware` needs to be updated, so that one can consume the changes. Please add also a `patch` changeset for `preview-middleware`.

Reply:
Changed in https://github.com/SAP/open-ux-tools/pull/1477/commits/a895920aa27a2912c029d8b26489bda248775584